test/python/picongpu/quick/picmi/distribution.py

```diff
@@ -448,3 +448,84 @@ def test_drift(self):
         self.assertAlmostEqual(drift.direction_normalized[0], 0.9370354841199405)
         self.assertAlmostEqual(drift.direction_normalized[1], 0.34920746753855203)
         self.assertAlmostEqual(drift.direction_normalized[2], 0.004318114799291135)
+
+
+class TestPicmiGaussianDistribution(unittest.TestCase, HelperTestPicmiBoundaries):
+    def _get_distribution(self, lower_bound=[None, None, None], upper_bound=[None, None, None]):
+        return picmi.GaussianDistribution(
+            density=42.42,
```

Comment:
Define as global variable (or so) in order to ensure testing against the same values later on.